airbyte-integrations/connectors/destination-pgvector/pyproject.toml

```diff
@@ -0,0 +1,86 @@
+[build-system]
+requires = [ "poetry-core>=1.0.0",]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "airbyte-destination-pgvector"
+version = "0.1.0"
+description = "Airbyte destination implementation for PGVector."
+authors = ["Airbyte <contact@airbyte.io>"]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/destinations/pgvector"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+
+[[tool.poetry.packages]]
+include = "destination_pgvector"
+
+[tool.poetry.dependencies.airbyte-cdk]
+# Version cnstrained by PyAirbyte (`airbyte`) version
+version = ">=1.0"
+extras = ["vector-db-based"]
```

Comment:
Is there any reason not to use the latest (^5) version of CDK?

I can see that pyairbyte uses 4.6.2 https://github.com/airbytehq/PyAirbyte/blob/main/pyproject.toml#L18

Reply:
Latest (^5) version of CDK Pandas would be a problem as 2.2.0 introduced some change that broke SQLAlchemy interop. 

We still can <5.0.0 CDK (see the below image), but not sure if is worth making an effort now till the pandas issue is resolved to have the latest CDK as there is still some stuff to fix with these versions that didn't conflict and we wish to ship a version of the connector soon. I could add this to the brainstorm list to work on subsequent iterations.

![image](https://github.com/user-attachments/assets/e88df5fa-844b-4758-be26-c36c455df3f7)
![image](https://github.com/user-attachments/assets/77f58fef-4d15-41be-869e-9835a1b8d23f)
![image](https://github.com/user-attachments/assets/d80c8139-38f9-4381-b992-064c01a531d0)

Reply:
And also something to do in [destination-snowflake-cortex](https://github.com/airbytehq/airbyte/blob/58d241d8eabe383bbcbc02b402f7b582d842de40/airbyte-integrations/connectors/destination-snowflake-cortex/pyproject.toml#L20-L28) (our base for pgvector) when we have time to pick this up.